src/main/cpp/t04_count.cpp

```diff
@@ -26,7 +26,23 @@
 
 
 using namespace std;
-
+int Word(char * S)
+{
+int D=1;
+    if (*S == NULL)
+    {
+        D=0;
+    }
+    while(*S)
+    {
+       if (*S == ' ')
+       {D++;}
+    *S++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [In expression like '*A++' the result of '*' is unused. Did you intend to write '(*A)++;'?](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4442729)